frontend/src/components/House/HouseList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
`import React, { Component} from 'react';`

Then you can `export default class HouseList extends Component {`
I've only just started doing this but realised I think it looks nicer. But take this comment as you want. Don't feel the need to change.

Reply:
ok, 2:1 :)